### ISSUE TITLE
Pop up notification if deleting files/folders failed.

### ISSUE THREAD

**Issue by kimlimjustin** [files-community/Files#3681]
**Is your feature request related to a problem? Please describe.**
When I permanently delete a `node_modules` folder, there's an error (I know there's an error since I try to delete it using File Explorer again). However, Files doesn't show anything when failed to delete (permanently) files/folders.

**Describe the solution you'd like**
A Pop-up error shows error(s) while deleting files/folders.

**Comment by gave92** [comment]
So you're deleting a folder with Files but at the end the folder is still there right?
Could you check if am error is reported in Files "Status Center" (icon in the lower right corner of the app)?

**Comment by kimlimjustin** [comment]
No, it keeps staying on deleting file 0%

![image](https://user-images.githubusercontent.com/61906684/110058609-3227cb80-7d95-11eb-8f19-c8b9fa85175e.png)

Debug file:
[debug_fulltrust.txt](https://github.com/files-community/Files/files/6087776/debug_fulltrust.txt)

**Comment by gave92** [comment]
Ah do not trust that progress, being that node_modules has a lot of files and given that Files is quite slow when deleting files it's probably still deleting xD

**Comment by kimlimjustin** [comment]
Deleting to recycle bin, I know that node_modules need a long time, but I let it delete for minutes and no result, when I try to delete using File Explorer, it shows me an error that operation not permitted, so I wonder that could Files give an error popup or notification that says there's an error to not waste user time to wait on it.

Besides, I found another bug that when I right-click on node_modules folder on Files, the apps freeze or sometimes crash. Not every time but always, should I open an issue about this?

**Comment by gave92** [comment]
> Should I open an issue about this?

Yes please, posting both debug files as well.
Did you take notice whether when the app crashes on opening the context menu you get a crash notification or not?

**Comment by kimlimjustin** [comment]
When it crashes and automatically closes, yes, a notification showing that there's a bug that contributors not prepared before. However, it's more frequent to freeze the apps rather than crashes and automatically close. I'll add more information on the new issue.
